data/base/jeos/sle15/packages.yaml

```diff
@@ -0,0 +1,46 @@
+packages:
+  image:
+    jeos:
+      - at
+      - audit
+      - bc
+      - blog
+      - cifs-utils
+      - dosfstools
+      - dracut
+      - glibc-locale
+      - grub2
+      - name: grub2-x86_64-efi
+        arch: x86_64
```

Comment:
Is this by convention ? I wondered why e.g grub2 is just a list member when grub2-x86_64-efi is a list member that is a hash. I guess this is done to allow other attributes like arch: right ?

Reply:
Yes, I needed a way to optionally specify the arch flag, but I didn't like having the name tag everywhere, so I added the option to specify the package name as a simple string, and the schema has a check for that. It maybe looks a bit odd.

Reply:
I think it's obvious enough and nicer than inventing syntax such as `- grub2-x86_64-efi:x86_64`. This would make it look more consistent in the snippet but opens the door to other problems. The "odd" look does not bother me.

Reply:
ok